### PR TITLE
OLJ-140 ropc manage sign-in details page

### PR DESCRIPTION
**Pull request by lutnos**

### CODE REVIEW

File: app/uk/gov/hmrc/credentialmanagementfrontend/models/OlfgJourneyId.scala [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+ */
+
+package uk.gov.hmrc.credentialmanagementfrontend.models
+
+import play.api.libs.json.{Format, Json}
+import play.api.mvc.QueryStringBindable
+
+import java.util.UUID
+
+case class OlfgJourneyId(value: UUID) {
```
Comment (stevenmallen) [inline app/uk/gov/hmrc/credentialmanagementfrontend/models/OlfgJourneyId.scala:24]: I don't think this model will ever be needed - please can we remove?

File: app/uk/gov/hmrc/credentialmanagementfrontend/controllers/CredentialManagementController.scala [hunk 2]
```diff
@@ -16,21 +16,21 @@
...
-  helloWorldPage: HelloWorldPage)
-    extends FrontendController(mcc) {
+class CredentialManagementController @Inject()(mcc: MessagesControllerComponents,
+                                               signInDetailsPage: SignInDetailsPage)
+  extends FrontendController(mcc) with Logging {
 
-  val helloWorld: Action[AnyContent] = Action.async { implicit request =>
-    Future.successful(Ok(helloWorldPage()))
+  def signInDetails(olfgJourneyId: OlfgJourneyId): Action[AnyContent] = Action { implicit request =>
+    Ok(signInDetailsPage(olfgJourneyId))
```
Comment (stevenmallen) [inline app/uk/gov/hmrc/credentialmanagementfrontend/controllers/CredentialManagementController.scala:33]: We need some kind of global switch that makes this (and any other page) return 404 (or 501) in production, please.

File: app/uk/gov/hmrc/credentialmanagementfrontend/models/OlfgJourneyId.scala (hunk 1 above)
Comment (lutnos) [reply app/uk/gov/hmrc/credentialmanagementfrontend/models/OlfgJourneyId.scala]: Removed it

File: app/uk/gov/hmrc/credentialmanagementfrontend/controllers/CredentialManagementController.scala (hunk 2 above)
Comment (lutnos) [reply app/uk/gov/hmrc/credentialmanagementfrontend/controllers/CredentialManagementController.scala]: Have added switch

**Review by stevenmallen — APPROVED** [review]
LGTM

**Pull request merged**